docs/webservices/iqrdemonstration.rst

```diff
@@ -186,31 +184,3 @@ Let us assume the IQR session state was downloaded as ``monarch.IqrState``.
 The following command will train a classifier leveraging the descriptors labeled by the IQR session that was saved::
```

Comment:
Minor: Can we update the above config to use the `SkLearnLogisticRegression` classifier like in the containerized service default configs?